lambdas/handlers/token_handler.py

```diff
@@ -1,267 +1,60 @@
 import json
-import os
-import time
-import uuid
 
-import jwt
-from boto3.dynamodb.conditions import Key
-from botocore.exceptions import ClientError
 from enums.logging_app_interaction import LoggingAppInteraction
-from enums.repository_role import RepositoryRole
-from models.oidc_models import IdTokenClaimSet
-from oauthlib.oauth2 import WebApplicationClient
-from services.dynamo_service import DynamoDBService
-from services.ods_api_service import OdsApiService
-from services.oidc_service import OidcService
-from services.ssm_service import SSMService
-from services.token_handler_ssm_service import TokenHandlerSSMService
+from services.login_service import LoginService
 from utils.audit_logging_setup import LoggingService
 from utils.decorators.ensure_env_var import ensure_environment_variables
 from utils.decorators.override_error_check import override_error_check
 from utils.decorators.set_audit_arg import set_request_context_for_logging
-from utils.exceptions import (AuthorisationException,
-                              OrganisationNotFoundException,
-                              TooManyOrgsException)
+from utils.exceptions import AuthorisationException
 from utils.lambda_response import ApiGatewayResponse
 from utils.request_context import request_context
 
 logger = LoggingService(__name__)
 
-token_handler_ssm_service = TokenHandlerSSMService()
-
 
 @set_request_context_for_logging
 @override_error_check
 @ensure_environment_variables(
     names=["AUTH_STATE_TABLE_NAME", "AUTH_SESSION_TABLE_NAME"]
 )
 def lambda_handler(event, context):
+    login_service = LoginService()
```

Comment:
Because this LoginService class implements a dynamo and ssm service, this initialisation should happen in the try/except in case any of them throw a ClientError